jekyll/_cci2/orb-author-faq.md

```diff
@@ -29,6 +29,18 @@ Error: Unable to find organization YOUR_ORG_NAME of vcs-type GITHUB: Must have m
 
 Read more in the [Orb CLI Permissions Matrix]({{site.baseurl}}/2.0/orb-author-intro/#permissions-matrix).
 
+## Deleting Orbs
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 23. You can fix by inserting `{: #deleting-orbs }` to the next line

Reply:
Added to all headers 👍 thank you. LMK how this looks.